### DISCUSSION TITLE
Welcome to the contribute section of discussions 👋

### DISCUSSION THREAD

**Discussion by joebudi** [Budibase/budibase#6789] [Contributing]
For users who are interested in contributing to Budibase, we're here to help. Please feel free to ask questions, suggest ideas, offer feedback and post any contributions you've made! Cheers Budis!

**Comment by JayP718** [comment]
Hi,

We are interested in contributing to budibase. We are looking to build a customized map marker ability and are looking to follow the design principles of budibase as well as a good starting point for incorporating this feature within the embedded map component. Looking forward to collaborate :)

**Comment by poirazis** [comment]
Hey @joebudi !
I too am interested in contributing by bringing some more Spectrum Webcomponents to BB. 
My first attempt would be to bring the Tabs Component and if successful bring more UI elements ( Buttons with Icons, Progressbars and more )

**Comment by joebudi** [comment]
Hey! It's wonderful to know there is an appetite from users to contribute back to Budibase. We are beginning to plan work for custom components and custom integrations, including the developer experience. @JayP718 and @poirazis, would you be interested in a quick call to dig deeper into the type of experience you'd expect when contributing?

**Reply by JayP718** [reply]
Absolutely, we already have some ideas 😁

**Reply by joebudi** [reply]
Tagging @shogunpurple for visibility

**Reply by joebudi** [reply]
Hey @JayP718 

It would be good to jump on a quick call sometime to discuss our plans around contributing custom integrations/components. Here's my Calendly link:
https://calendly.com/budibase/

Please book a slot.
Joe

**Reply by vevedh** [reply]
Hi, do you speak French ?

**Comment by poirazis** [comment]
Yeap @joebudi , I am in to share any thoughts !

**Reply by joebudi** [reply]
Hey @poirazis 
It would be good to jump on a quick call sometime to discuss our plans around contributing custom integrations/components. Here's my Calendly link:
https://calendly.com/budibase/
Joe

**Reply by vevedh** [reply]
Hey, @joebudi 
Do you speak french ? little ?
